Add the  Deployment and Detailed Status (#3839)
MicrosoftDocs/windows-itpro-docs: windows/deployment/update/update-compliance-security-update-status.md

 
The **Latest Security Update Status** and **Previous Security Update Status** tiles are stacked to form one blade. The **Latest Security Update Status** provides a visualization of the different deployment states devices are in regarding the latest update for each build (or version) of Windows 10, along with the revision of that update. The **Previous Security Update Status** blade provides the same information without the accompanying visualization. 

> The deployment and detailed states of an update, are described as follows:

Reviewer: ```suggestion
The deployment and detailed states of an update are described as follows:
```

**Deployment States**

> * **Unknown:** Occurs when devices are not scanned for an update in some time.

Reviewer 2: ```suggestion
* **Unknown** occurs when devices have not been scanned for an update for some time.
```

> * **Unknown:** Occurs when devices are not scanned for an update in some time.

Reviewer 2: ```suggestion
* **Unknown**: Occurs when devices have not been scanned for an update for some time.
```

> * **In Progress:** When an update is applying to a specific device.

Reviewer 2: ```suggestion
* **In Progress** means an update is being applied to a specific device.
```

> * **In Progress:** When an update is applying to a specific device.

Reviewer 2: ```suggestion
* **In Progress**: When an update is being applied to a specific device.
```

> * **Progress Stalled:** When for any reason, the progress of an update stops, for example caused by netowork problems.

Reviewer 2: ```suggestion
* **Progress Stalled**: When the progress of an update stops for any reason, such as network failure.
```

> * **Failed:** When an update, failed to apply to a specific device.

Reviewer 2: ```suggestion
* **Failed**: When the update process for a specific device is unsuccessful.
```

> * **Update Completed:** When an update applies to a device without problems.

Reviewer 2: ```suggestion
* **Update Completed**: When a device is updated successfully.
```

> * **Deferred:** When the application of an update for a device, it is deferred for another time. For example postponing the application one day after planned. 

Reviewer 2: ```suggestion
* **Deferred**: A device update has been postponed to a later time. 
```

> * **Cancelled:** When the applying of an update is cancelled by the user.

Reviewer 2: ```suggestion
* **Cancelled**: Indicates that an update has been cancelled by the user.
```

**Detailed States**

> * **Dowload Started:** Indicates that downloading the contents of a specific update started.

Reviewer 2: ```suggestion
* **Download Started**: Indicates that a specific update has started to download.
```

> * **Download Succeeded:** Indicates that downloading the content of a specific update ended successfully.

Reviewer 2: ```suggestion
* **Download Succeeded**: Indicates that the download of a specific update has been successful.
```

> * **Download Hand Off:** Indicates that you can continue downloading an update at a point in time where it was stopped.

Reviewer 2: ```suggestion
* **Download Hand Off**: Indicates that you can continue downloading an update at the point where it was stopped.
```

> * **Install Hand Off:** Indicates that you can continue installing an update at a point in time where it was stopped.

Reviewer 2: ```suggestion
* **Install Hand Off**: Indicates that you can continue installing an update at the point where it was stopped.
```

> * **Reboot Pending:** Indicates that an update needs a reboot to finish applying.

Reviewer 2: ```suggestion
* **Reboot Pending**: Indicates that you need reboot the device to complete the update process.
```

Reviewer: ```suggestion
* **Reboot Pending**: Indicates that you need to reboot the device to complete the update process.
```

> * **Service Stack Downloaded:** Indicates that the Service Stack Update (The component that installs Windows update) is downloading.

Reviewer 2: ```suggestion
* **Service Stack Downloaded**: Indicates that the Servicing Stack Update (the component that installs the Windows update) is downloading.
```

> * **Update Offered:** Indicates that an update is offered, being optional the application of the same.

Reviewer 2: ```suggestion
* **Update Offered**: Indicates that an optional update is available.
```

> * **Update Detected:** Indicates that a needed update is detected to apply.

Reviewer 2: ```suggestion
* **Update Detected**: Indicates that a necessary update is available.
```
